cypress/e2e/models/migration/task-manager/task-manager.ts

```diff
@@ -77,4 +82,16 @@ export class TaskManager {
         click(searchButton);
         cy.wait(2 * SEC);
     }
+
+    public static enablePreemption(rowName: string): void {
+        TaskManager.open(10);
+        cy.contains(rowName)
+            .next(tdTag)
+            .contains("Pending")
```

Comment:
There are more kind of statuses like Ready/Planing/Quota blocked

Can we check if the status is not in (running, failed, succeeded), IIRC the documentation mentioned those three for when the user cannot set preemption

Reply:
As of now UI only shows "Running" "Succeeded" and "Pending" tasks . 
Running and succeeded can't be preempted.
Only pending can be . 

For other tasks we wil have to see how to generate those tasks .